### PR TITLE
#184581326 Setup docker-compose

### PR DESCRIPTION
**Pull request by INValentin**
### What does this PR do?
Setup docker-compose to run backend and database applications

### Description of Task to be completed

- Installing docker/docker-compose
- Creating docker images for node backend & postgres database applications
- Making sure the node backend can connect to postgres database

### installed packages
- sequelize (for testing db connection)

### How should this be manually tested
- Install [docker](https://www.docker.com/products/docker-desktop/) & docker compose
- `cd to/project/directory` (after cloning)
- Add `DATABASE_URL` to `.env` file for now you may give it this value `DATABASE_URL=postgres://admin:admin123@database:5432/ecommerce`
- `docker build -t node-backend .`
- `docker-compose up`

**If you see "Database connected successfully", it's working!**

### Any background context you want to provide
- You may change database credentials from `docker-compose.yml` file in `environment` section
### Related pivot story id
https://www.pivotaltracker.com/n/projects/2632830/stories/184581326

### CODE REVIEW

**Comment by Patienceineza** [comment]
LGTM

**Review by bencyubahiro77 — APPROVED** [review]
everything is fine

**Review by edwinnambaje — CHANGES_REQUESTED** [review]
I think the title of your PR is lacking the pivot tracker id

**Review by blackd44 — APPROVED** [review]
Better to me

**Review by bencyubahiro77 — APPROVED** [review]

**Comment by nshutijonathan** [comment]
> I think the title of your PR is lacking the pivot tracker id

He should rename his PR the way it is supposed to be.

**Review by Zeden12 — APPROVED** [review]
LGTM

**Comment by INValentin** [comment]
> > I think the title of your PR is lacking the pivot tracker id
> 
> He should rename his PR the way it is supposed to be.

My bad title fixed!

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Hey @INValentin 
Squash your commits into one and resolve conflicts

File: .env.example [hunk 1]
```diff
@@ -3,4 +3,13 @@ PORT =
 DATABASE_URL = postgres://{user}:{password}@{hostname}:{port}/{database-name}
 DEV_DATABASE_URL = postgres://{user}:{password}@{hostname}:{port}/{database-name}
 
-SWAGGER_SERVER = http://localhost:<your port>
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (nshutijonathan) [inline .env.example]: Hey 

These are conflicts pushed ,fix them

**Review by nshutijonathan — CHANGES_REQUESTED** [review]

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Pull request merged**